testapp/ng1/alt_root_index.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>My AngularJS App</title>
+  <link rel="stylesheet" href="app.css"/>
+</head>
+<body>
+
+  <div id="outside-ng">
+    {{1 + 2}}
+  </div>
+
+  <div id="nested-ng-app" ng-app="myApp">
+    <div id="inside-ng">
+      {{1 + 2}}
+    </div>
+
+    <ul class="menu">
+      <li><a href="#/repeater">repeater</a></li>
```

Comment:
We can clean this up quite a bit and just have one page that does async stuff. I'm OK doing that in a follow-up PR.